src/components/ViewDao/DaoSidebar.tsx

```diff
@@ -30,7 +30,17 @@ class DaoSidebarComponent extends React.Component<IProps, null> {
               <b className={css.daoIcon} style={{backgroundImage: bgPattern.toDataUrl()}}></b>
               <span>{dao.name}</span>
             </Link>
-            <p>Anyone can make a proposal to the DAO! Click the button on the top right.</p>
+          </div>
+          <div className={css.daoDescription}>
+            { dao.name === "Meme" ?
```

Comment:
This is really very very ugly... :-/ we should consider having a registry, liek for the schemes ....

Reply:
let;'s merge it anyway and backlog https://github.com/daostack/alchemy/issues/751

Reply:
yes of course, I thought we are already planning a big team conversation about a DAO registry